docs/platforms/javascript/guides/bun/index.mdx

```diff
@@ -53,22 +55,17 @@ Sentry.init({
   // https://docs.sentry.io/platforms/javascript/configuration/options/#traces-sample-rate
   tracesSampleRate: 1.0,
   // ___PRODUCT_OPTION_END___ performance
+  // ___PRODUCT_OPTION_START___ logs
+
+  // Enable logs to be sent to Sentry
+  enableLogs: true,
+  // ___PRODUCT_OPTION_END___ logs
 });
 ```
 
-Once you set a `tracesSampleRate`, performance instrumentation is automatically enabled for you. See <PlatformLink to="/tracing/instrumentation/automatic-instrumentation">Automatic Instrumentation</PlatformLink> to learn about all the things that the SDK automatically instruments for you.
-
-You can also manually capture performance data - see <PlatformLink to="/tracing/instrumentation/custom-instrumentation">Custom Instrumentation</PlatformLink> for details.
-
-<PlatformContent includePath="getting-started-sourcemaps" />
-
-## Use
+### Apply Instrumentation to Your App
 
-<Alert level='warning'>
-
-Import `instrument.js` before any other modules to ensure Sentry initializes early. If you initialize later, auto-instrumentation and modules like database monitoring, agent monitoring, tracing may not work.
-
-</Alert>
+Import `instrument.js` before any other modules to make sure Sentry initializes early. If you initialize later, auto-instrumentation and modules like tracing may not work.
```

Comment:
Bun has the same problem as Node with ESM (in Node, we have the "Installation methods"). In bun we would need to `--preload` this instrumentation file to be loaded before everything else: https://bun.com/docs/runtime#param-preload

This is out of scope for this PR but I just want to mention that. We need to improve our docs to be less Node-focused and also include other runtimes such as Bun and Deno in the future.